thirdparty/xmlstream/xmlstream_p.h

```diff
@@ -1140,10 +1140,11 @@ bool XmlStreamReaderPrivate::parse()
             case '\n':
                 ++lineNumber;
                 lastLineStart = characterOffset + readBufferPos;
+                [[gnu::fallthrough]];
```

Comment:
It seems the rest is accepted by gcc as apparently intentional. It only complains about this case.